### PR TITLE
Feature/df 2061 multibranch for jenkins

### PR DESCRIPTION
**Pull request by cinhtau**
- add branch conditionals
- parallel docker builds (speed up pipeline)

### CODE REVIEW

File: openshift/job-ad-service/jenkinsfile.groovy [hunk 1]
```diff
@@ -125,87 +111,123 @@ pipeline {
...
-
-                            def build = openshift.selector('bc', 'app-job-ad-service-docker').startBuild("--from-dir .")
-                            result = build.logs('-f')
-
-                            if(result.status == 0) {
-                                return true
-                            }
+                rtMavenRun(
+                        pom: 'pom.xml',
+                        goals: 'sonar:sonar -Dsonar.projectKey=ReferenceService -Dsonar.host.url="$SONAR_SERVER" -Dsonar.login=$SONAR_LOGIN',
```
Comment (fabiangotzen) [inline openshift/job-ad-service/jenkinsfile.groovy]: -Dsonar.projectKey=ReferenceService -> JobadService? (I don't know the exact project key)

Comment (cinhtau) [reply openshift/job-ad-service/jenkinsfile.groovy]: Oops, gonna fix that

Comment (seco-mgabor) [reply openshift/job-ad-service/jenkinsfile.groovy]: Yep,that's the name of the project that will be created in SonarQube.

Comment (cinhtau) [reply openshift/job-ad-service/jenkinsfile.groovy]: changed

**Pull request merged**